examples/step-29/step-29.cc

```diff
@@ -940,6 +940,8 @@ int main ()
       using namespace dealii;
       using namespace Step29;
 
+      deallog.depth_console(5);
```

Comment:
> Why so?

because step-29 explicitly writes to ``deallog`` and otherwise doesn't print anything.